### ISSUE TITLE
Model Name change causes catastrophic failure

### ISSUE THREAD

**Issue by davidgs** [strongloop/strong-arc#1715]
I had models named Accelerometer-X, Accelerometer-Y, etc. Turns out the - is not a great character to use in model names, as in other places, it is an ‘illegal character’. It isn’t flagged by composer as such though. So I went in to the composer and tried to change the names of the models to remove the - character. As soon as I removed it from one model, ALL of my models disappeared from the composer and I now have NO models in the composer, and can no longer run the application. I’ve tried to find all the instances of the - in all the files in the project manually, but so far, I’ve had no luck. The application is dead, and I can't seem to resuscitate it.
 
Basically, renaming a model has caused the entire project to be unusable. Seems like a bug, no?

**Comment by p1lare** [comment]
Similar in here. In my case renaming the name of model in Composer ended up with not showing up any other Model in  composer, but since i had it open I've deleted the model, and after restarting arc it was back on. I could add the model with the name I wanted.
